src/composables/multisigAccountCreate.ts

```diff
@@ -0,0 +1,146 @@
+import { computed, ref } from '@vue/composition-api';
+import BigNumber from 'bignumber.js';
+import { AmountFormatter, Crypto, TxBuilder } from '@aeternity/aepp-sdk';
+import type {
+  IDefaultComposableOptions,
+  IKeyPair,
+  IMultisigAccountBase,
+  IMultisigCreationPhase,
+} from '../types';
+import { useSdk } from './sdk';
+import { MULTISIG_CREATION_PHASES, MULTISIG_SIMPLE_GA_BYTECODE } from '../popup/utils';
+import SimpleGAMultiSigAci from '../lib/contracts/SimpleGAMultiSigACI.json';
+
+export function useMultisigAccountCreate({ store }: IDefaultComposableOptions) {
+  const { getDrySdk, getSdk } = useSdk({ store });
+
+  let rawTx: string;
+  let accountId: string;
```

Comment:
why you declare this variables as "let" not as "ref"

Reply:
Because the reactivity is not needed here. This is a state shared between two function and it's not passed to any template/watcher.